### PR TITLE
#163418590 setup node environment

### PR DESCRIPTION
**Pull request by Chiazokam**
#### What does this PR do?
This PR sets up the node environment

#### Description of Task to be completed?
- set up eslint
- set up travis
- set up coveralls
- set up mocha and chai

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ch-setup-dev-env-163418590 `
- open the index.html file on the browser and navigate to the desired pages

#### What are the relevant pivotal tracker stories?
[#163418590](https://www.pivotaltracker.com/story/show/163418590)

##### Coveralls setup
image

#### Travis setup
image

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["env"]
```
Comment (JCanaks) [inline .babelrc:2]: Resolve all EOLs on this PR

File: .nyc_output/12c498d2-22f5-49fe-9ce9-b7715b02f18c.json [hunk 2]
```diff
@@ -0,0 +1 @@
..."column":13},"end":{"line":24,"column":14}},"loc":{"start":{"line":24,"column":33},"end":{"line":26,"column":1}},"line":24}},"branchMap":{"0":{"loc":{"start":{"line":15,"column":46},"end":{"line":15,"column":92}},"type":"cond-expr","locations":[{"start":{"line":15,"column":70},"end":{"line":15,"column":73}},{"start":{"line":15,"column":76},"end":{"line":15,"column":92}}],"line":15},"1":{"loc":{"start":{"line":15,"column":46},"end":{"line":15,"column":67}},"type":"binary-expr","locations":[{"start":{"line":15,"column":46},"end":{"line":15,"column":49}},{"start":{"line":15,"column":53},"end":{"line":15,"column":67}}],"line":15},"2":{"loc":{"start":{"line":31,"column":11},"end":{"line":31,"column":35}},"type":"binary-expr","locations":[{"start":{"line":31,"column":11},"end":{"line":31,"column":27}},{"start":{"line":31,"column":31},"end":{"line":31,"column":35}}],"line":31}},"s":{"0":1,"1":1,"2":1,"3":1,"4":1,"5":2,"6":1,"7":1,"8":1,"9":1,"10":1,"11":1,"12":1,"13":1,"14":1},"f":{"0":2,"1":1},"b":{"0":[0,2],"1":[2,2],"2":[1,1]},"_coverageSchema":"43e27e138ebf9cfc5966b082cf9a028302ed4184","hash":"64dd45a5fcb0df2aa8e3db3974111346b88385f7","contentHash":"3f65748b3b66be9626ce644e3747cabf"}}
```
Comment (JCanaks) [inline .nyc_output/12c498d2-22f5-49fe-9ce9-b7715b02f18c.json]: include this file as part of the things to git ignore and ensure that you remove tracking reference. Follow guide specified  here --> https://www.atlassian.com/git/tutorials/saving-changes/gitignoresoecified

**Comment by JCanaks** [comment]
@Chiazokam The feedback for this PR is yet to be implemented pls do so asap

**Comment by Chiazokam** [comment]
> @Chiazokam The feedback for this PR is yet to be implemented pls do so asap

Sorry, I can't see the feedback

**Review by JCanaks — CHANGES_REQUESTED** [review]

File: .nyc_output/12c498d2-22f5-49fe-9ce9-b7715b02f18c.json (hunk 2 above)
Comment (Chiazokam) [reply .nyc_output/12c498d2-22f5-49fe-9ce9-b7715b02f18c.json]: Alright...thanks

**Pull request closed without merging**